xdsl/backend/riscv/lowering/optimise_riscv.py

```diff
@@ -15,3 +15,14 @@ def match_and_rewrite(self, op: riscv.MVOp, rewriter: PatternRewriter) -> None:
             and op.rd.type.is_allocated
         ):
             rewriter.replace_matched_op([], [op.rs])
+
+
+class RemoveRedundantFMv(RewritePattern):
```

Comment:
`RemoveRedundantFMv` looks a little bit weird, expect `RemoveRedundantFmv` or `RemoveRedundantFMV` as class is `FMVOp`.

Reply:
I see your point, but I'm not sure it's strictly better. The tradeoff is between "the set of all optimisations for a specific operation are easy to find" and "the set of all optimisations for the riscv dialect are easy to find". I think having them all in `optimise_riscv.py` will help us going forward. We can always refactor if we decide that it's getting too noisy.